crates/contracts/build.rs

```diff
@@ -759,7 +836,7 @@ fn main() {
             .add_network_str(GOERLI, "0x1F98431c8aD98523631AE4a59f267346ea31F984")
             .add_network_str(SEPOLIA, "0x1F98431c8aD98523631AE4a59f267346ea31F984")
             .add_network_str(ARBITRUM_ONE, "0x1F98431c8aD98523631AE4a59f267346ea31F984")
-        // Not available on Gnosis Chain
+        // Not available on Gnosis Chain and Base
```

Comment:
Looks like the address is `0x33128a8fC17869897dcE68Ed026d694621f6FDfD` on base see [here](https://docs.uniswap.org/contracts/v3/reference/deployments/base-deployments).